src/features/events/tests/appPromotionCard.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { GroupedEventsCards, AppPromotionCard } from '../index'
+import { mockNodes, testEvent } from './__mocks__'
+import 'jest-styled-components'
+
+global.___loader = {
+  enqueue: jest.fn(),
+}
+
```

Comment:
@laij84 I this file as a tsx file because I saw you had the setup.ts configured for this but it didn't work for me. :/ suggestions? should I modify the setup.js configuration ?